### PR TITLE
Cursorless enablement group page

### PR DESCRIPTION
**Pull request by pokey**
See [deploy preview](https://deploy-preview-1660--cursorless.netlify.app/enablement-group)

- Also adds support for mdx in our top-level cursorless-org package

## Checklist

- [x] Add meta social tags
- [x] Use latest logo
- [x] Why is logo on social slow? And is it new logo?
- [x] Make prices clickable
- [x] document that sponsorship levels are yearly
- [-] I have added [tests](https://www.cursorless.org/docs/contributing/test-case-recorder/)
- [-] I have updated the [docs](https://github.com/cursorless-dev/cursorless/tree/main/docs) and [cheatsheet](https://github.com/cursorless-dev/cursorless/tree/main/cursorless-talon/src/cheatsheet)
- [-] I have not broken the cheatsheet

### CODE REVIEW

File: packages/cursorless-org/src/content/enablement-group.mdx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  title: "Cursorless Enablement Group",
+  description:
+    "Help enable the adoption of Cursorless by speeding up its development.",
+};
+
+# {meta.title}
+
+## 🚀 {meta.description}
+
+Cursorless, an open-source spoken programming language, lets users eliminate the need for a cursor entirely. In other words, software engineers can code at the speed of sound (increased productivity) and all computer users can reduce strain on their wrists to prevent injury (preventative healthcare).
```
Comment (pokey) [inline packages/cursorless-org/src/content/enablement-group.mdx:11]: I'll wordsmith this one.

File: packages/cursorless-org/src/content/enablement-group.mdx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+- Ability to promote your organization as a leader in the area of accessible technology.
+- Opportunity to connect with other influencers in software and accessibility.
+- Networking and ability to cultivate deep relationships and potential partnerships with leaders, founders, and influencers.
+
+#### 🪖 Strategic
+
+- Support an initiative targeted at facilitating an increased and more rapid shift of adoption of Cursorless as a faster way to code and a healthier way to code.
+- Support of open standards approach. Standards are critical to interoperability and openness is consistent with a positive image in the industry.
+
+{/* <!-- Individual, Nonprofit, and corporate tiers go here. I do not believe I can stylize them well --> */}
```
Comment (pokey) [inline packages/cursorless-org/src/content/enablement-group.mdx:66]: did you want me to style these as boxes like in your Figma? @F-Kunkle

File: packages/cursorless-org/src/content/enablement-group.mdx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+Cursorless needs a dedicated, full-time software engineer on staff. This will cost $5,000 USD per month. The Enablement Group guides development and the participation fees contribute to this crucial funding. Cursorless will always be open source.
+
+## 🙌 Join the Cursorless Enablement Group
+
+The Enablement Group consists of stakeholders whose talents, lived experiences, and career experiences bring vital perspectives to the development of Cursorless. Support and input help increase the speed at which new features can be delivered.
+
+## 🥇 Benefits of Joining the Cursorless Enablement Group
+
+{/* <!--- Hey Pokey! Can you do a bit of styling for these 5 which will consider the major benefits. I have ideas in the figmaa wireframe if helpful! --> */}
```
Comment (pokey) [inline packages/cursorless-org/src/content/enablement-group.mdx:31]: I do think even with the image / emojis we could use a bit more visual interest. Shall I take a crack at making these into boxes like the Figma, or really doesn't matter?

**Review by F-Kunkle — APPROVED** [review]
Hey both of these edits look grat!

File: packages/cursorless-org/src/content/enablement-group.mdx (hunk 1 above)
Comment (F-Kunkle) [reply packages/cursorless-org/src/content/enablement-group.mdx:11]: Sounds good! Are you OK with launching as it is or plan to wordsmith prior to launcing?

File: packages/cursorless-org/src/content/enablement-group.mdx (hunk 3 above)
Comment (F-Kunkle) [reply packages/cursorless-org/src/content/enablement-group.mdx:31]: @pokey hey - I do think you could stylize if you have the time. Whether you use the Figma wireframe or have a way to make them "stand out" more with more common markdown code, open to that.
I'll say that getting the program launched soon is attractive to me at this point

**Review by F-Kunkle — APPROVED** [review]
I like all of these edits and think they're an improvement! Reminder that GitHub is still a bit new to me so i'm hoping i'm approving the 5 edits I saw on this page in a fashion that works for you. Thx!

**Comment by pokey** [comment]
ok @AndreasArvidsson this one is good to go

**Comment by AndreasArvidsson** [comment]
I had no idea that this concept was this far along. Do we need to talk about some aspects of this or do you just want to run this all yourself?

This doesn't look to good on firefox
image

**Comment by pokey** [comment]
There's a marketing guy who's been managing the whole thing; happy to chat tho if you want more detail on where things are

That's supposed to be cut off like that as an indication that you can scroll horizontally. I'll see if I can come up with another way to indicate that that's less confusing 🤔

**Comment by AndreasArvidsson** [comment]
Gladly!

There is no scroll bar or carousel arrows. Not the most intuitive ux ;)

**Review by AndreasArvidsson — APPROVED** [review]

**Comment by pokey** [comment]
ok @AndreasArvidsson I decided to use your patented KISS system and just put them directly in the flow of the document 😄

**Pull request merged**